### DISCUSSION TITLE
Alpha 150

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4310] [Release Notes]
* You can now rearrange split panes by holding `Cmd-Shift` or `Ctrl-Shift` and dragging them around:

<img width="941" alt="Screenshot 2021-07-31 at 19 08 13" src="https://user-images.githubusercontent.com/161476/127747375-e2170ae0-73ae-496a-a8d4-6921ab70928a.png">

* Split tab title now combines titles of all panes
* Fixed config file getting spontaneously erased
* Hotkeys can now consist of modifier keys only

# Changes from Alpha 149 included in this release

* You can now drag other tabs into the current tab to attach them as split panes
* Profiles: option to limit or disable "recent profiles"
* SFTP: transfers are now written into a temporary file first, and then moved into place when the transfer is complete
* SFTP: the panel (and WinSCP) will open in the current directory if [configured](https://github.com/Eugeny/tabby/wiki/Shell-working-directory-reporting)
* SFTP: tabby will prevent OS from sleeping during transfers
* SSH: make sure keepalive is enabled by default

<img src="https://ga-beacon.appspot.com/UA-3278102-18/github/release-150" width="1"/>

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.150'>Alpha 150</a>.</em>

**Comment by SynapseScribe** [comment]
1. how to move back to tabbar a tab after moving it in current tab (with CTRL key) ?
2. Which is the difference between 'dragging into current tab' or 'attaching as split pane' ?

**Reply by Eugeny** [reply]
1. There's currently no way to turn panes back into separate tabs
2. It's the same thing
